src/app/providers/storage/strategies/__mocks__/storage.provider.ts

```diff
@@ -0,0 +1,5 @@
+export class StorageMockClass {
+  get() { }
+
+  set() { }
```

Comment:
return something dummy

Reply:
What should I return in set method?? In fact, I'm no returning anything

Reply:
nothing, really sorry about that